### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Jolak5**
In this PR, I

1. refactored my HTML code
2. used CSS to hide some elements
3. used javascript to create the responsive hamburger

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi  @Jolak5, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the correct linter configurations, this project uses Javascript. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -16,17 +16,20 @@
...
+
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#section-1">Portfolio</a></li>
+            <li class="nav-item"><a href="#section-2">About</a></li>
+            <li class="nav-item"><a href="#section-3">Contact</a></li>
+          </ul>
+
+          <div class="hamburger">
+            <img src="images/Icon - Menu.svg" alt="Hamburger menu" />
+          </div>
```
Comment (GzimAsani) [inline index.html]: - [ ] When the hamburger is clicked it looks different than on design, kindly handle this:
Yours:
image
Desing:
image

Comment (GzimAsani) [reply index.html]: - [ ] Welcome should  be present only on mobile
image

Comment (GzimAsani) [reply index.html]: - [ ] After I open the hamburger menu and click on any of the options, they send me to the correct part of the page, but after that, I cannot move on your app anymore, kindly check that

Comment (GzimAsani) [reply index.html]: - [ ] According to the design the welcome and hamburger button should be on the same line
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @Jolak5                             :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project, hamburger button works as expected :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Jolak5 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**